modules/common/src/main/MarkdownRender.scala

```diff
@@ -208,6 +209,10 @@ object MarkdownRender:
       else
         val link         = context.resolveLink(LinkType.LINK, node.getUrl().unescape(), null, null)
         def justAsLink() = renderLinkWithBase(node, context, html, link)
+
```

Comment:
scala code should not use the `return` keyword because sometimes it can have bizarre effects.  But never fear - the last expression in a scope is the implicit return value - the keyword is just there due to java heritage).  See my edits

Reply:
I see, I was learning Scala on my own time, and I read something about that but it went way over my head 😄 . Thank you for the edit.